pkg/apis/logging/v1alpha1/fluentbit_types.go

```diff
@@ -44,6 +44,7 @@ type FluentbitSpec struct {
 type FluentbitTLS struct {
 	Enabled    bool   `json:"enabled"`
 	SecretName string `json:"secretName"`
+	SecretType string `json:"secretType"`
```

Comment:
I suggest to let this be omitted for backwards compatibility
```suggestion
	SecretType string `json:"secretType,omitempty"`
```